### PR TITLE
Update font style 

### LINKED ISSUE #5730: Disc Number is not displayed in properties of Music file
### Description

The Disc Number property is not being displayed in Music file properties.

### Steps To Reproduce

1. Right click on a Music file 
2. Click on "Properties" in the context menu
3. Check file details, Disc Number does not show up

### Expected behavior

I expect to see the disc number in the file properties.

### Files Version

v1.9.20.0

### Windows Version

Windows 11 21H2 22000.120

### Relevant Assets

image

This is what shows up in the file properties

### PR DESCRIPTION
**Pull request by winston-de**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #5730
- https://github.com/files-community/Files/pull/5717#discussion_r684726948

**Details of Changes**
Add details of changes here.
- Updated font style in details layout, grid view, and preview pane to use caption text style
- added file tag to preview pane
- Fix issue where right clicking text in rename box would commit rename
- add part of set property
- fix issue where tab item would not be selected on load
- fix alignment issue with tags in tiles view

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml.cs [hunk 1]
```diff
@@ -224,6 +225,16 @@ public Visibility TabStripVisibility
...
 
+        private bool firstLoad = true;
 
+        private void TabViewItem_Loaded(object sender, RoutedEventArgs e)
+        {
+            // fixes issue where tab would not show as selected when opened with path argument
+            if(firstLoad)
+            {
+                firstLoad = false;
+                (sender as TabViewItem).IsSelected = true;
```
Comment (gave92) [inline Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml.cs:236]: Would `(sender as TabViewItem).Loaded -= TabViewItem_Loaded` work instead of using the `firstLoad` thing?

File: Files/ViewModels/ItemViewModel.cs [hunk 2]
```diff
@@ -816,7 +816,9 @@ private async Task LoadItemThumbnail(ListedItem item, uint thumbnailSize = 20, I
                         var matchingStorageFolder = (StorageFolder)matchingStorageItem ?? await GetFolderFromPathAsync(item.ItemPath);
                         if (matchingStorageFolder != null)
                         {
-                            using var Thumbnail = await matchingStorageFolder.GetThumbnailAsync(ThumbnailMode.ListView, thumbnailSize, ThumbnailOptions.ReturnOnlyIfCached);
+                            var mode = thumbnailSize < 80 ? ThumbnailMode.ListView : ThumbnailMode.SingleItem;
+
+                            using var Thumbnail = await matchingStorageFolder.GetThumbnailAsync(mode, thumbnailSize, ThumbnailOptions.UseCurrentScale);
```
Comment (gave92) [inline Files/ViewModels/ItemViewModel.cs:821]: Completely unrelated: could someone explain me the difference between `UseCurrentScale` and `ResizeThumbail` options? When should I use one or the other?

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml.cs (hunk 1 above)
Comment (winston-de) [reply Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml.cs:236]: I don't think so, the event would have to be removed from all TabViewItems since the event is in the template.

Comment (gave92) [reply Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml.cs:236]: Sorry I had misunderstood your intentions here.

File: Files/ViewModels/ItemViewModel.cs (hunk 2 above)
Comment (winston-de) [reply Files/ViewModels/ItemViewModel.cs:821]: My understanding is that resize thumbnail resizes the thumbnail (up or down) to match the requested size, while increase size just scales up the size to match the scaling of the display.

Comment (gave92) [reply Files/ViewModels/ItemViewModel.cs:821]: So `UseCurrentScale` should be used all the times?

Comment (winston-de) [reply Files/ViewModels/ItemViewModel.cs:821]: Yeah I don't think it would make much of a difference.

Comment (gave92) [reply Files/ViewModels/ItemViewModel.cs:821]: Thanks. I believe we do the opposite in most other places xD

File: Files/Views/LayoutModes/DetailsLayoutBrowser.xaml [hunk 3]
```diff
@@ -815,18 +801,26 @@
...
                                                                 <ItemsPresenter
                                                                     Padding="{TemplateBinding Padding}"
                                                                     HorizontalAlignment="Stretch"
-                                                                    Footer="{TemplateBinding Footer}"
                                                                     FooterTemplate="{TemplateBinding FooterTemplate}"
                                                                     FooterTransitions="{TemplateBinding FooterTransitions}"
                                                                     HeaderTemplate="{TemplateBinding HeaderTemplate}"
                                                                     HeaderTransitions="{TemplateBinding HeaderTransitions}" />
+
+                                                                <ContentPresenter
```
Comment (gave92) [inline Files/Views/LayoutModes/DetailsLayoutBrowser.xaml:820]: Why the change? The footer now appears left aligned instead of center-aligned, was it intentional?

File: Files/Views/LayoutModes/DetailsLayoutBrowser.xaml [hunk 4]
```diff
@@ -150,6 +150,16 @@
...
             </Style>
+
+            <Style
+                x:Key="ColumnContentTextBlock"
+                BasedOn="{StaticResource CaptionTextBlockStyle}"
+                TargetType="TextBlock">
+                <Setter Property="TextWrapping" Value="NoWrap" />
+                <Setter Property="TextTrimming" Value="CharacterEllipsis" />
+                <Setter Property="VerticalAlignment" Value="Center" />
+                <Setter Property="HorizontalAlignment" Value="Stretch" />
```
Comment (gave92) [inline Files/Views/LayoutModes/DetailsLayoutBrowser.xaml:161]: What about keeping the `FontWeight="Light"` or add `Opacity="0.6"` like in TilesView?

File: Files/Views/LayoutModes/DetailsLayoutBrowser.xaml [hunk 5]
```diff
@@ -150,6 +150,16 @@
             <Style TargetType="uc:DataGridHeader">
                 <Setter Property="HorizontalContentAlignment" Value="Stretch" />
             </Style>
+
+            <Style
+                x:Key="ColumnContentTextBlock"
+                BasedOn="{StaticResource CaptionTextBlockStyle}"
```
Comment (gave92) [inline Files/Views/LayoutModes/DetailsLayoutBrowser.xaml:156]: I'm not sold to this change in DetailsView: CaptionTextBlockStyle looks _really_ small and seeing two different font sizes on the same row does not look good to me, but I will bow to the designers opinion.. @yair100? xD

File: Files/Views/LayoutModes/DetailsLayoutBrowser.xaml (hunk 3 above)
Comment (winston-de) [reply Files/Views/LayoutModes/DetailsLayoutBrowser.xaml:820]: Should be fixed in 649e004

File: Files/Views/LayoutModes/DetailsLayoutBrowser.xaml (hunk 5 above)
Comment (winston-de) [reply Files/Views/LayoutModes/DetailsLayoutBrowser.xaml:156]: Windows 11 introduced Segoe UI variable font which is much more legible at small sizes then old segoe ui. This change is intended to leverage that. Perhaps the old style should be used on windows 10? 

Variable vs standard at size 12:
image

Comment (yaichenbaum) [reply Files/Views/LayoutModes/DetailsLayoutBrowser.xaml:156]: @gave92 I don't have any preference here so I think we should probably use what the design specs suggest.

File: Files/Views/LayoutModes/DetailsLayoutBrowser.xaml (hunk 4 above)
Comment (winston-de) [reply Files/Views/LayoutModes/DetailsLayoutBrowser.xaml:161]: Good idea

Comment (winston-de) [reply Files/Views/LayoutModes/DetailsLayoutBrowser.xaml:161]: image

**Review by gave92 — APPROVED** [review]
LGTM
